mitxgraders/baseclasses.py

```diff
@@ -402,11 +459,6 @@ def check(self, answers, student_input, **kwargs):
                    "Expected at least one answer in answers")
             raise ConfigError(msg)
 
-        # Make sure the input is in the expected format
```

Comment:
Is this now caught by `ensure_text_inputs`?

Reply:
Yes. 

All graders now run `ensure_text_inputs` during `__call__`, which unicode or list of unicode. This is one of the few structural changes to the library in this PR. Without this change, I believe that each grader class would "need" to separately coerce input strings to unicode during `check`.

(I say "need" because it's really only an issue for local testing of library or local testing of course problem codes. edX gives unicode, so nothing on edX should ever really need that coercion. )